include/boost/wave/util/flex_string.hpp

```diff
@@ -549,7 +549,11 @@ class AllocatorStringStorage : public A
 
     void* Alloc(size_type sz, const void* p = 0)
     {
+#if defined(BOOST_NO_CXX11_ALLOCATOR)
         return A::allocate(1 + (sz - 1) / sizeof(E),
+#else
+        return std::allocator_traits<A>::allocate(*this, 1 + (sz - 1) / sizeof(E),
+#endif
             static_cast<const char*>(p));
```

Comment:
Please duplicate this line inside both branches of the #if/#else.

Reply:
Done

Reply:
Thanks a lot!